File: Java/Ejercicio3.java
import javax.swing.JOptionPane;

public class Ejercicio3 {
    public static void main(String[] args) {
        int anio = Integer.parseInt
        (JOptionPane.showInputDialog("Año del vehículo:"));
        int tipo = Integer.parseInt
        (JOptionPane.showInputDialog("Tipo (1=Particular, 2=Carga, 3=Público):"));
        int emisiones = Integer.parseInt
        (JOptionPane.showInputDialog("Nivel de emisiones:"));

        String resultado = "Verificación rechazada";
        if (tipo == 1 && emisiones <= 50) resultado = "Verificación aprobada";
        else if (tipo == 2 && emisiones <= 70) resultado = "Verificación aprobada";
        else if (tipo == 3 && emisiones <= 60) resultado = "Verificación aprobada";
        String advertencia = "";
        if (anio < 2005) advertencia = "\nADVERTENCIA: Vehículo antiguo, requiere revisión especial";

        JOptionPane.showMessageDialog(null, resultado + advertencia);
    }
}

File: 3-Python/1-Programas/1-EstructurasSimples/ejercicio3.py
anio = int(input("Año del vehículo: "))
tipo = int(input("Tipo (1=Particular, 2=Carga, 3=Público): "))
emisiones = int(input("Nivel de emisiones: "))

resultado = "Verificación rechazada" # [cite: 61]

# Reglas [cite: 58-60]
if tipo == 1 and emisiones <= 50:
    resultado = "Verificación aprobada"
elif tipo == 2 and emisiones <= 70:
    resultado = "Verificación aprobada"
elif tipo == 3 and emisiones <= 60:
    resultado = "Verificación aprobada"

print(f"Resultado: {resultado}")
if anio < 2005:
    print("Vehículo antiguo, requiere revisión especial") # [cite: 64]